platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -134,6 +134,11 @@ It will be removed in Gradle 10.0.
 
 Since Gradle 8.5, the same information can be obtained via the `BuildFeatures` service using `configurationCache.requested` property.
 
+[[undeprecated_configuration_uasge]]
```

Comment:
🤔 As with #33005, I think it makes sense to add a note to the original deprecation

Reply:
+1

Reply:
> I think we should also make these proper admonition blocks, using NOTE:

Done for the comment from #33005.